components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -378,6 +378,14 @@ class RewardsServiceImpl : public RewardsService,
   void OnURLLoaderComplete(SimpleURLLoaderList::iterator url_loader_it,
                            ledger::client::LoadURLCallback callback,
                            std::unique_ptr<std::string> response_body);
+  void OnURLLoaderCompleteSanitizeJson(
+      ledger::client::LoadURLCallback callback,
+      ledger::mojom::UrlResponse response,
+      std::unique_ptr<std::string> response_body);
+  void OnURLLoaderCompleteSanitizeJsonAndroid(
```

Comment:
We can delete this now, I think?

Reply:
Sorry, I have missed this. Fixed at 
https://github.com/brave/brave-core/pull/17354/files#diff-df0abf3b2c26fef46c7ac0d8b8d6e22d344e8b69576dcf33faed8c595b1cdc0dR607